### PR TITLE
Add support for MacOSX storyboards

### PR DESCRIPTION
**Pull request by phimage**
I have added the support of MacOSX storyboards by 
- adding an enum 'OS', which provide strings for output
- reading OS in storyboard files
- moving the code at the end of files in 'processStoryboards' function

I make some test on my two different MacOSX Storyboards (but maybe I don't use all features)
and check that there is no output differences after my modification on NatalieExample

### CODE REVIEW

File: natalie.swift [hunk 1]
```diff
@@ -555,6 +555,125 @@ extension XMLElement: Printable {
...
+    }
+    
+    var storyboardType: String {
+        switch self {
+        case iOS: return "UIStoryboard"
+        case OSX: return "NSStoryboard"
+        }
+    }
+    
+    var storyboardTypeUnwrap: String {
```
Comment (krzyzanowskim) [inline natalie.swift:606]: why this is a thing at all ?

Comment (phimage) [inline natalie.swift:606]: for iOS : init(name name: String,
   bundle storyboardBundleOrNil: NSBundle?) -> UIStoryboard

for MacOSX : convenience init**?**(name name: String,
           bundle storyboardBundleOrNil: NSBundle?)
This init() must be unwrapped

**Comment by krzyzanowskim** [comment]
uderstand. Thank you. Very good work! I was completly unaware of OSX in that regard.

 Ill merge this one, and do some refactoring later. I think some parts can be simplified for incrqsed readability for this new dual state. 

Wondering how long we'll be able to keep both of these under the same umbrella though :-)

**Comment by phimage** [comment]
Yes some OS variables like storyboardTypeUnwrap are not clear
Maybe sometimes instead of delegate to os:OS (os.XXX), some 'if' or 'switch' could be used directly
if os == OS.iOS { println(something) }
else if os == OS.MacOS { println(something else) }

**Pull request merged**